### ISSUE TITLE
getPlugins silently fails on require errors

### ISSUE THREAD

**Issue by andy-hanson** [lukeed/taskr#87]
[`getPlugins`](https://github.com/flyjs/fly/blob/master/src/cli/spawn.js#L25) responds to errors in `require` by calling `alert` and continuing.

But [`alert`](https://github.com/flyjs/fly-util/blob/master/src/logging/index.js#L39) will only do anything when the environment variable VERBOSE is set.

`getPlugins` will then silently fail and return `undefined`.

This eventually leads to a failure [here](https://github.com/flyjs/fly/blob/master/src/fly.js#L37):
`TypeError: Cannot read property 'call' of undefined`.

**Comment by ghost** [comment]
@andy-hanson That's right! Great catch. I actually added a fix for this already and is shipping in 0.8.0, here is the snippet:

``` js
    plugins.forEach(({ name, plugin }) => {
      if (!plugin) throw new Error(`Did you forget to npm i -D ${name}?`)
      plugin.call(this, debug(name.replace("-", ":")), _("load %o", name))
    })
```

But if you have a suggestion or better idea now is the time to let me know :smile:

**Comment by ghost** [comment]
:airplane:  080e84bf348a3b3d485dd78e28e615062748d5ae
